core/domain/summary_services.py

```diff
@@ -23,6 +23,17 @@
 import utils
 
 _LIBRARY_INDEX_GROUPS = [{
+    'header': 'Recently Published',
```

Comment:
Don't add a group here. See how featured explorations etc. are dealt with, and follow that instead. These are just the category-specific groups.